test/CodeGenOpenCL/is_valid_event.cl

```diff
@@ -0,0 +1,16 @@
+// Make sure that OpenCL builtins declared in opencl.h are translated
```

Comment:
should we have a separate directory test/CodeGenSPIRV ? It would be easier for upstreaming.